activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/TaskProvider.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps;
+
+import java.util.List;
+
+import org.activiti.api.task.model.Task;
+
+public interface TaskProvider {
```

Comment:
@erdemedeiros same here.. TaskSource ?? really up to you.. but we need to be careful because these interfaces are now part of our APIs

Reply:
As for events, I like `TaskSource`! I'll update it.